rest/route/project_test.go

```diff
@@ -140,7 +137,9 @@ func (s *ProjectPatchByIDSuite) TestRunWithValidBbConfig() {
 	s.NotNil(resp)
 	s.NotNil(resp.Data())
 	s.Require().Equal(http.StatusOK, resp.Status())
-	s.Require().Equal("EVG", s.sc.MockProjectConnector.CachedProjects[0].BuildBaronSettings.TicketCreateProject)
+	pRef, err := s.sc.DBProjectConnector.FindProjectById("dimoxinil", false, false)
```

Comment:
Do we need to specify the connector type? i think s.sc.FindProjectById should be fine

Reply:
True, removing